doc/api.yml

```diff
@@ -1272,6 +1394,37 @@ components:
       enum:
         - ENABLED
         - DISABLED
+    StudentMissing:
+      type: object
+      properties:
+        id:
+          type: string
+        pointing_datetime:
+          type: string
+          format: date-time
+        place:
+          type: string
+        is_late:
+          type: boolean
+        is_missing:
+          type: boolean
+        calendar:
+          $ref: '#/components/schemas/Calendar'
+        student:
+          $ref: '#/components/schemas/Student'
+    Calendar:
```

Comment:
Is this Calendar object already have a resource to handle it ?

Reply:
I already feat the calendar resources in the teaching api.